apps/dashboard/src/components/activity/components/activity-overview.tsx

```diff
@@ -37,12 +38,18 @@ export function ActivityOverview({ activity }: ActivityOverviewProps) {
 
         <OverviewItem label="Transaction ID" value={activity.transactionId} isCopyable />
 
-        <OverviewItem
-          label="Subscriber ID"
-          isDeleted={!activity.subscriber}
-          value={(activity.subscriber?.subscriberId || activity._subscriberId) ?? ''}
-          isCopyable
-        />
+        <SubscriberDrawerButton
+          className="text-start"
+          subscriberId={activity.subscriber?.subscriberId || activity._subscriberId}
+          readOnly={true}
+        >
+          <OverviewItem
+            label="Subscriber ID"
+            isDeleted={!activity.subscriber}
+            value={(activity.subscriber?.subscriberId || activity._subscriberId) ?? ''}
+            isCopyable
+          />
```

Comment:
We should handle the case when the subscriber is deleted; in that case, IMO, we should show the `Deleted` as the first name and last name, but let's clarify that with Radek or Sok. And also we should not show the `Preferences` tab.

https://github.com/user-attachments/assets/53556f32-f7a7-4a56-8031-c116d5a5f219